### PR TITLE
Implementation of "last updated" for both metadata, and in page display

### PR DESCRIPTION
**Pull request by dannychapman**
- Added sample last update to YAML front matter for Disability benefits
> conditions content page
- Added last update as meta data using RDFa implementation of Dublin
core
- Added to content page template for 1-topic-land as sample.

WIP do not merge

### CODE REVIEW

**Comment by dannychapman** [comment]
This is in reference to: https://github.com/department-of-veterans-affairs/vets.gov-content/issues/57#issuecomment-271613352

File: content/includes/header.html
Comment (carlmjohnson) [inline content/includes/header.html]: BOM? Stray whitespace?

File: content/includes/header.html
Comment (carlmjohnson) [inline content/includes/header.html]: I think the format for this is supposed to be `<meta content="{{lastupdate}}" name="DC.Date" >`.

File: content/pages/disability-benefits/conditions.md
Comment (carlmjohnson) [inline content/pages/disability-benefits/conditions.md]: What format is this date supposed to be in? My assumption is that it should be YYYY-MM-DD, but maybe not.

**Comment by rroueche** [comment]
@dannychapman 2 questions:

1) @maryannbrody would like the full month name to be spelled out: November instead of Nov. Is this possible given the tools you're using?

File: content/includes/header.html
Comment (dannychapman) [inline content/includes/header.html]: Added

File: content/pages/disability-benefits/conditions.md
Comment (dannychapman) [inline content/pages/disability-benefits/conditions.md]: Flipped order, seems to work ok.

**Comment by dannychapman** [comment]
@rroueche 

1. I've been trying to get the full month to spell out, but no luck yet. @carlmjohnson do you have time to chip in on this?
2. For this sample, I've added it to the YAML at the top of the conditions page - take a look there on this branch to see the format. 
3. Adding the date to the YAML front matter on any page will automatically add it to the bottom in plain text. (Once this is merged into master).

**Comment by rroueche** [comment]
Thanks @dannychapman !

More on 2. : will the editor need to manually update the date here: `lastupdate: 2016-11-24` in the YAML when they change something on a page? Or if it's dynamic, what does that date pull from?

More on 3. : we don't want it to show up on all pages, so do we need to go in manually and add that YAML element to any page for which we'd like to show the data? Seems like we'd want that metadata for allll pages in the site, but might only want it to actually show up on the content-heavy ones. cc'ing @maryannbrody if you have thoughts/input on this.

**Comment by dannychapman** [comment]
2: Yes on manual. Nothing happens unless it is entered. 
3. It will only show up on pages where that particular bit of front matter is present.

**Comment by rroueche** [comment]
2. Ok so we'll just need to be careful about making sure we update that date, and also use the correct format.

3. I think we'd probably want to collect this data for all pages in the site, for internal analytics purposes, but wouldn't want it visually displayed on some pages, like the home page for example. BUT I don't know if either of those statements are true - what do you think? slash who else's opinion should we ask on that?

Also, maybe that's future state since the more immediate goal is just having the date show at all (which is done!)

**Comment by dannychapman** [comment]
@rroueche I think we should revive the auto-date / internal display only option as we get deeper into content tooling discovery.

**Comment by dannychapman** [comment]
This PR adds metadata and "Last updated" text to the bottom of a content page, of the associated YAML front matter is present. To view a sample, visit: disability benefits > conditions.

**Comment by jkassemi** [comment]
This should run successfully after a master merge. Jenkins is running on branch builds now, and this includes a deprecated Jenkinsfile that skips branch-based builds in favor of pull request builds. 

No rush to resolve if it's not a priority to get this content out, this just came up as part of the investigation in https://github.com/department-of-veterans-affairs/devops/issues/1352

**Comment by maryannbrody** [comment]
This looks good to me! Tagging @emilyville for situational awareness!

**Comment by rroueche** [comment]
@dannychapman this is ready to move on to next steps. Can we go ahead and get it into the 'merge to master' process, or are there other technical checks it needs to go through?

**Comment by rroueche** [comment]
@dannychapman checking back in on this - any ideas on when this can be merged in and ready for us to start using?

**Comment by dannychapman** [comment]
@rroueche I'd like it to have a quick review by both @melwoodard and someone from USDS if possible.

**Comment by rroueche** [comment]
@melwoodard and @crwallace do you have time in the next couple of days to review?

**Review by melwoodard — COMMENTED** [review]
@dannychapman is there a reason why the date is bold? I'm worried that makes it seem too significant, and could cause concern about whether certain content is accurate. I'd keep all the text at regular weight unless there's a strong argument to emphasize the date.

**Comment by dannychapman** [comment]
@melwoodard Good call on the time stamp. It certainly could have a negative effect. Unbolded!

**Review by melwoodard — APPROVED** [review]
LGTM - @crwallace can you take a look at this today?

**Comment by crwallace** [comment]
This looks good and we can merge it in so you have something for now but if you create a GH ticket and assign it to me I can add `metalsmith-update` so you won't have to manually update the date every time the file changes.

**Review by crwallace — APPROVED** [review]

**Comment by rroueche** [comment]
awesome, thanks @crwallace ! will do.

there are some travis errors that look like they have to do w/broken code -- sample:
image who's the best person to take a look at that?

Once that's resolved, then I should be able to merge.

**Comment by dannychapman** [comment]
@rroueche @carlmjohnson  is taking a look at the errors.

**Comment by rroueche** [comment]
ok perfect thanks!

**Comment by rroueche** [comment]
@carlmjohnson yay! looks like everything is good now. did you want to make any more updates before I merge?

**Pull request merged**
